app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -55,7 +59,12 @@ class HomeFragment : Fragment() {
     }
 
     private fun initTiles() = with (tileContainer) {
-        adapter = HomeTileAdapter(onUrlEnteredListener)
+        var homeTiles = mutableListOf<JSONObject>()
+        val jsonArray = HomeTileUtils.loadDefaultTiles(context).getJSONArray("default_tiles")
```

Comment:
nit: pull this out into a constant

In general, you don't want to hard-code strings into the file because 1) it makes it more error prone if you use it elsewhere, and 2) people have to know what string to search for, and where it's used, rather than just being able to see it at the top of the file as a constant.